docs/02_database/01_fields-tables-views/02_views/01_views-basics.mdx

```diff
@@ -70,6 +70,8 @@ The following example shows three key things:
 
 ```
 
+It is worth noting that if you are working with real-time data in your views, you probably need to use [backwards joins](#backwards-joins), which we shall look at later in this page.
```

Comment:
This shouldn't be the default setting for views in data servers. It should only be used when the joined data needs to be updated in real time.